agent/consul/config.go

```diff
@@ -448,9 +448,12 @@ func DefaultConfig() *Config {
 	// Disable shutdown on removal
 	conf.RaftConfig.ShutdownOnRemove = false
 
-	// Check every 5 seconds to see if there are enough new entries for a snapshot
+	// Check every 5 seconds to see if there are enough new entries for a snapshot, can be overridden
 	conf.RaftConfig.SnapshotInterval = 5 * time.Second
```

Comment:
I wonder if we should bump this back up again to something like 30-60s just to make things a little nicer in the case where a busy cluster hasn't configured this and gets really frequent snapshots.

Reply:
Plan was to set this to 30 seconds and the threshold to 16384